### ISSUE TITLE
Users App - Synchronise changes in memberships on the Wizard tab via server events

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp-apps#434]
Case 1:
1. Open the Role Wizard for any role
2. Open the User Wizard in another tab (keep the first one open) and add the same role to this user
3. Switch back to the first tab with the Role Wizard
BUG: new added user is not displayed in 'members
![role-w](https://cloud.githubusercontent.com/assets/3728712/23752827/f6bae020-04e7-11e7-8b4a-441582c09fba.png)

Case 2:
Same as above but with the Group wizard

Case 3:
Same actions as above but add user either in the Role Wizard or the Group Wizard - changes will not be reflected in the Wizard for that user.

Basically, we should implement update of the Wizard's membership lists upon receiving server update events.

**Comment by runarmyklebust** [comment]
This is not possible to handle this by using the node.updated events:

If Role-wiz open, and user changed (by adding role to user), the role-tab will update since the role is modified
If user-tab is open, and user is added from role-tab, the user-tab will not be updated since the user-node is never changed (which makes sense)

This is really a very specific scenario for the UserApp, and no changes should be done to server events to support this I think? Is it possible to send an application internal event to make sure the user-tab is updated?
